index.html

```diff
@@ -0,0 +1,526 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My Florence</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./assets/bootstrap/bootstrap.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:300|Poppins:300,600" rel="stylesheet">
+    <link href="./public/stylesheets/style.css" rel="stylesheet">
+  </head>
+
+  <body>
+  <div id="main">
+
+    <!--1) GREEN BACKGROUND START-->
+    <div class="bg-green">
+
+    <!--HEADER-->
+      <div class="content-wrap lrg-rl-padding" id="header">
```

Comment:
I like using the sectioning tag for the main wrapper whenever possible. In this case, I'd suggest:

```html
<header class="content-wrap lrg-rl-padding" id="header">
```

And then use a `<h1>` for the logo, like:

```html
<h1>
   A Viget Exploration
   <img class="header-img" src="./public/img/title.png" alt="Improving Medical Adherence">
</h1>
```